mne/viz/_3d.py

```diff
@@ -1143,3 +1143,107 @@ def plot_dipole_locations(dipoles, trans, subject, subjects_dir=None,
         fig.scene.x_plus_view()
 
     return fig
+
+
+def snapshot_montage(fig, montage, with_points=False):
```

Comment:
you could use it with any montage + mayavi figure. It doesn't map sensors onto the pial surface, all it does is take the 3d coordinates in the montage and project them onto the 2d plane of the current viewing camera. Then it also returns the screenshot of the current camera view.